src/Polly.RateLimiting/RateLimiterResilienceStrategy.cs

```diff
@@ -65,7 +65,8 @@ protected override async ValueTask<Outcome<TResult>> ExecuteCore<TResult, TState
             await OnLeaseRejected(new OnRateLimiterRejectedArguments(context, lease)).ConfigureAwait(context.ContinueOnCapturedContext);
         }
 
-        var exception = retryAfter.HasValue ? new RateLimiterRejectedException(retryAfter.Value) : new RateLimiterRejectedException();
+        var source = _telemetry.GetTelemetrySource();
+        var exception = retryAfter.HasValue ? new RateLimiterRejectedException(source, retryAfter.Value) : new RateLimiterRejectedException(source);
```

Comment:
Needs changing to use the new constructors? Otherwise `source` is being used as the message.

Reply:
:tired_face: :tired_face: :tired_face: